packages/web/src/scss/components/Accordion/index.html

```diff
@@ -0,0 +1,188 @@
+{{#> layout/plain }}
+
+<section class="docs-Section">
+
+  <h2 class="docs-Heading">Default accordion (Always open)</h2>
+
+  <div class="mb-400">
+
+    <section
+      class="Accordion"
+      data-toggle="accordion"
+    >
+
+      <!-- Accordion Item -->
+      <article
+        id="accordionExample_article_0"
+        class="Accordion__item"
+      >
+        <h3
+          id="accordionExample_article_0_header"
+          class="Accordion__itemHeader"
+        >
+          <button
+            type="button"
+            class="Accordion__itemToggle"
+            data-toggle="collapse"
+            data-target="accordionExample_article_0_collapse"
+          >
+            <!-- toggle -->
+            Accordion header #0
+            <!-- // toggle -->
+          </button>
+          <span class="Accordion__itemSlot">
+            <!-- slot -->
+            <a href="#">Link</a>
+            <span class="Pill Pill--selected">3</span>
+            <svg class="Accordion__itemIcon" width="24" height="24" aria-hidden="true">
```

Comment:
So you decided not to wrap the icon in `Accordion__itemIcon`?

Reply:
Nope, i wrapped it in branch with JS, i must change it in here too.